lib/starter/builder/templates/activerecord.rb

```diff
@@ -0,0 +1,117 @@
+# frozen_string_literal: true
+
+module Starter
+  module Templates
+    module ActiveRecord
+      def model_klass
+        'ActiveRecord::Base'
+      end
+
+      def initializer
+        <<-FILE.strip_heredoc
+        require 'yaml'
+        require 'erb'
+
+        config_dir = File.expand_path('../', __FILE__)
+        config_content = File.read(File.join(config_dir, 'database.yml'))
```

Comment:
change to: `config_content = File.read(File.join('config', 'database.yml'))` and remove Line#15